llvm/lib/Target/AArch64/AArch64StackTagging.cpp

```diff
@@ -578,6 +578,8 @@ bool AArch64StackTagging::runOnFunction(Function &Fn) {
     // Fixup debug intrinsics to point to the new alloca.
     for (auto *DVI : Info.DbgVariableIntrinsics)
       DVI->replaceVariableLocationOp(OldAI, Info.AI);
+    for (auto *DPV : Info.DbgVariableRecords)
```

Comment:
This seems untested. I can comment this out and 

```
build/bin/llvm-lit -v llvm/test/CodeGen/AArch64/
```

still passes.

Reply:
It's not tested under the default configuration and so the lack-of-failure is expected. To ensure the code isn't rotting, we've got this [0] buildbot set up to exercise the "new debug-info" configuration, which requires passing `LLVM_EXPERIMENTAL_DEBUGINFO_ITERATORS=On` to cmake. With that enabled, the test RUN lines where `--try-experimental-debuginfo-iterators` has been added will operate under "new" debug-info mode, and  you Should (TM) see a test failure.

We're inches away from turning this on-by-default and it being tested everywhere, after which there won't be this obscure testing arrangement happening any more.

[0] https://lab.llvm.org/buildbot/#/builders/275

Reply:
Oh thanks! I'll keep that in mind. I wanted to change this code a bit so wanted to make sure I'm actually running the tests, now I know how.

Reply:
I re-rean this with `-DLLVM_EXPERIMENTAL_DEBUGINFO_ITERATORS=On` in the CMake but it still passes if I comment out these lines.